include/runtime/header.h

```diff
@@ -25,11 +25,11 @@
 #include <runtime/collections/rangemap.h>
 #include <unordered_set>
 
-struct MatchLog {
+struct match_log {
   enum { SUCCESS = 0, FUNCTION, FAIL } kind;
```

Comment:
Again, shouldn't this be `lower_case` based on `readability-identifier-naming.EnumCase`? Or `CamelCase` based on `readability-identifier-naming.EnumConstantCase`? Not sure which one applies in this case.

Reply:
As above, it's `EnumConstantCase` - should be `CamelCased`